README.md

```diff
@@ -1,64 +1,104 @@
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=alert_status)](https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi) [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi) [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi) [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=coverage)](https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi) [![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi) 
```

Comment:
Although I like the fact that these are placed first now (shows that we intend to have a codebase of great quality!) I feel like it's optically out of place this way. How about moving it between line 17/18, with a container that has a max-width?

Reply:
Also, you've used shorthands on line 65/66. I like that, didn't know that was a thing. I'd suggest using it here as well.

Reply:
I've implemented the shorthand, however it does not seem to work withing a div.
So I guess we have to pick which we want. I lean to just jamming all the badges on one for now and keeping the shorthand. let me know what you think.